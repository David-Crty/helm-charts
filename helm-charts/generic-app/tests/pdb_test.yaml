# yaml-language-server: $schema=https://raw.githubusercontent.com/helm-unittest/helm-unittest/main/schema/helm-testsuite.json

suite: test ingress
release:
  name: test
chart:
  version: 0.1.0
templates:
  - pdb.yaml
tests:
  - it: should have pod disruption budget
    set:
      pdb:
        enabled: true
    asserts:
      - matchSnapshot: {}