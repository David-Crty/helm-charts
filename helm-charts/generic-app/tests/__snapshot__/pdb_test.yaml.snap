should have pod disruption budget:
  1: |
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:
      labels:
        app.kubernetes.io/instance: test
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: generic-app
        helm.sh/chart: generic-app-0.1.0
      name: test-generic-app
    spec:
      minAvailable: 1
      selector:
        matchLabels:
          app.kubernetes.io/component: web
          app.kubernetes.io/instance: test
          app.kubernetes.io/name: generic-app
